src/contexts/contexts.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const screenWidthContext = React.createContext({
+  screenWidth: 0,
+});
+
+const searchResultsContext = React.createContext({
+  searchResults: [],

```

Comment:
`screenWidthContext` and `searchResultsContext` should be in **UpperCase**, because the context variablse act as `components` in the markup